task/runner.go

```diff
@@ -471,6 +471,36 @@ func errorInfo(err error) *data.ErrorInfo {
 	return &data.ErrorInfo{Message: err.Error(), Unretriable: IsUnretriable(err)}
 }
 
+// Caller should check if err is a CatalystError first
+func humanizeCatalystError(err error) error {
+	errMsg := strings.ToLower(err.Error())
+
+	// General errors
+	if strings.Contains(errMsg, "import request") && strings.Contains(errMsg, "504 Gateway Timeout") {
+		return errors.New("file could not be imported from URL because it was not accessible")
+	}
+
+	// MediaConvert pipeline errors
+	if strings.Contains(errMsg, "doesn't have video that the transcoder can consume") {
+		// TODO(yondonfu): Add link in this error message to a page with the input codec/container support matrix
+		return errors.New("invalid video file codec or container, check your input file against the input codec and container support matrix")
+	} else if strings.Contains(errMsg, "Failed probe/open") {
```

Comment:
Notice that the error message is lowercased, so it will never match this. Needs to match against an all lowercase string instead.

Edit: realized that this came as a debt from the previous code that did these checks. I guess that has never worked 😯